### PR TITLE
i18n(zh-cn): Update docs about synced-tabs

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description

- Update docs about synced-tabs
- https://github.com/withastro/starlight/pull/640

### CODE REVIEW

**Review by huyikai — COMMENTED** [review]
LGTM.

File: docs/src/content/docs/zh-cn/guides/customization.mdx [hunk 1]
```diff
@@ -162,7 +162,7 @@ defineConfig({
 
 通过将 `tableOfContents` 选项设置为 `false` 来完全禁用目录：
 
-<Tabs>
+<Tabs syncKey="config-type">
   <TabItem label=" frontmatter ">
```
Comment (HiDeoo) [inline docs/src/content/docs/zh-cn/guides/customization.mdx:166]:
```suggestion
  <TabItem label="Frontmatter ">
```

To make sure the label are exactly identical and sync properly.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the update, left a small suggestion to fix a sync issue but other than that, looks perfect to me 👍

**Review by HiDeoo — APPROVED** [review]
Thanks for the fast update :thumbsup:

**Pull request merged**